src/Core/Cache/TraitCompareDelete.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Friendica\Core\Cache;
+
+use Friendica\Core\Cache;
+
+/**
+ * Trait TraitCompareSetDelete
+ *
+ * @brief This Trait is to compensate non native "exclusive" sets/deletes in caches
+ *
+ * @package Friendica\Core\Cache
+ */
+trait TraitCompareDelete
+{
+	abstract public function get($key);
```

Comment:
Can’t you just refer to the `IMemoryCacheDriver` interface instead of duplicating the methods prototypes?

Reply:
A trait cannot implement interfaces